mne/decoding/tests/test_csp.py

```diff
@@ -201,3 +201,25 @@ def test_ajd():
                 [0.694689013234610, 0.775690358505945, -1.162043086446043],
                 [-0.592603135588066, -0.598996925696260, 1.009550086271192]]
     assert_array_almost_equal(V, V_matlab)
+
+
+def test_spoc():
+    X = np.random.randn(10, 10, 20)
+    y = np.random.randn(10)
+
+    spoc = SPoC(n_components=4)
+    spoc.fit(X, y)
+    Xt = spoc.transform(X)
+    assert_array_equal(Xt.shape, [10, 4])
+    spoc = SPoC(n_components=4, transform_into='csp_space')
+    spoc.fit(X, y)
+    Xt = spoc.transform(X)
+    assert_array_equal(Xt.shape, [10, 4, 20])
+    assert_array_equal(spoc.filters_.shape, [10, 10])
+    assert_array_equal(spoc.patterns_.shape, [10, 10])
```

Comment:
We have code for simulation since we were originally working on an example with  simulated data.
If i can make it compact enough for the test, i will push it.